README.md

```diff
@@ -1,15 +1,5 @@
-# JavaScript Conditionals
-
-## Objectives
-1. Explain what constitutes an expression in JavaScript.
-2. Organize code using block statements.
-3. Describe the difference between truthy and falsy values.
-4. Employ `if...else` statements to conditionally execute blocks of code.
-5. Use a `switch` statement to selectively execute code based on the value of
-an expression.
-6. Rewrite an `if...else` as a one-liner with the ternary operator.
-
-## Overview
+# JavaScript Conditionals 
+## Problem Statement 
 In all facets of life, we constantly make conditional choices:
```

Comment:
I like this as an intro. I wonder if it would be more effective if it were first in English, then the code version.